offchain/hello-world-browser/src/Page/Home.purs

```diff
@@ -85,12 +92,29 @@ component =
 
   handleAction :: forall slots. Action -> H.HalogenM State Action slots o m Unit
   handleAction = case _ of
+    Resume -> do
+      H.modify_ $ const Resuming
+      result <- resume helloWorldIncrement
+      case result of
+        Left err -> do
+          H.modify_ $ const (LockFailed err)
+        Right funds -> do
+          new <- getDatum
+          case new of
+            Left err -> H.modify_ $ const $ (ResumeFailed err)
+            Right new' -> H.modify_ $ const (Locked new' funds)
```

Comment:
NITPICK :)

```suggestion
          getDatum >>= case _ of
            Left err -> H.modify_ $ const (ResumeFailed err)
            Right new -> H.modify_ $ const (Locked new funds)
```

Reply:
Thanks for reviewing so carefully. Can't believe I just forgot how to \case between the two fixes :facepalm:.